bool write_e_stop
int8 write_gear
int16 write_speed
int16 write_brake
int32 write_steer